content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -101,161 +101,180 @@ data:
 EOF
 {{< /text >}}
 
-## Adding services from other clusters
+## Setup steps
```

Comment:
These setup steps don't really seem appropriate in this document, they seem like they should be in a separate example document. This document would then just need a line of text that says something like:

To experiment with an actual multicluster configuration of this type, see our [Gateway-based multicluster example](/docs/examples/multicluster/new-example/).